src/pages/rewards/RewardsPage.tsx

```diff
@@ -1,17 +1,326 @@
+                  params: { FROM: <strong>Ethereum</strong>, TO: <strong>dYdX Chain</strong> },
+                })}
+              </span>
+            </Styled.MobileMigrateHeader>
+          }
+        >
+          <Styled.WithReceipt
+            slotReceipt={
+              <Styled.Details
+                items={[
+                  {
+                    key: 'available-to-migrate',
+                    label: stringGetter({ key: STRING_KEYS.AVAILABLE_TO_MIGRATE }),
+                    value: <Output type={OutputType.Asset} value={tokenBalance} />,
+                  },
+                ]}
+              />
+            }
+          >
+            <Button
+              action={tokenBalance ? ButtonAction.Primary : ButtonAction.Base}
+              type={ButtonType.Link}
+              href={BRIDGE_URL}
+              size={ButtonSize.Medium}
+              disabled
+            >
+              {tokenBalance
+                ? `${stringGetter({ key: STRING_KEYS.MIGRATE_NOW })} →`
+                : stringGetter({ key: STRING_KEYS.NO_TOKENS_TO_MIGRATE })}
+            </Button>
+          </Styled.WithReceipt>
+          <Styled.LearnMore>
+            {stringGetter({ key: STRING_KEYS.WANT_TO_LEARN })}
+            <Link href={MIGRATE_HELP_URL}>{stringGetter({ key: STRING_KEYS.CLICK_HERE })}</Link>
+          </Styled.LearnMore>
+        </Styled.MobileMigrateCard>
+      )}
+
+      <Styled.PanelRow>
+        {isTablet && (
+          <Styled.BalancePanelContainer>
+            <DYDXBalancePanel />
+          </Styled.BalancePanelContainer>
+        )}
+
+        <Styled.Panel
+          slotHeader={<Styled.Title>{stringGetter({ key: STRING_KEYS.GOVERNANCE })}</Styled.Title>}
+        >
+          <Styled.Row>
+            <Styled.Description>
+              {stringGetter({ key: STRING_KEYS.GOVERNANCE_DESCRIPTION })}
+              <Link href={GOVERNANCE_HELP_URL}>
+                {stringGetter({ key: STRING_KEYS.LEARN_MORE })} →
+              </Link>
+            </Styled.Description>
+            <IconButton
+              action={ButtonAction.Base}
+              iconName={IconName.Arrow}
+              onClick={() => dispatch(openDialog({ type: DialogTypes.Keplr }))}
+              size={ButtonSize.Small}
+            />
+          </Styled.Row>
+        </Styled.Panel>
+
+        <Styled.Panel
+          slotHeader={<Styled.Title>{stringGetter({ key: STRING_KEYS.STAKING })}</Styled.Title>}
+        >
+          <Styled.Row>
+            <Styled.Description>
+              {stringGetter({ key: STRING_KEYS.STAKING_DESCRIPTION })}
+              <Link href={STAKING_HELP_URL}>{stringGetter({ key: STRING_KEYS.LEARN_MORE })} →</Link>
+            </Styled.Description>
+            <IconButton
+              action={ButtonAction.Base}
+              iconName={IconName.Arrow}
+              onClick={() => dispatch(openDialog({ type: DialogTypes.Keplr }))}
+              size={ButtonSize.Small}
+            />
```

Comment:
nit -- but is it possible to make this vertically center of the whole panel?
<img width="358" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/1380ecea-2ce4-4cc3-a639-f85e8deb1d47">
<img width="754" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/7314ad91-40c9-4587-adf3-82d2edbf46e1">

Reply:
also small nit: add back border to icon button + make arrow darker color

Reply:
apparently the whole panel is clickable too hehe, feel free to add a TODO here for later polish (center vertically + probably can bring this into the Panel component itself, + make the whole thing clickable)